spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
You made some mistake in the expansion, the `init` form is actually in the `require` form right before calling the `config`.
Here is the expansion I've got:

```elisp
(progn
  nil
  (eval-when-compile
    (when (bound-and-true-p byte-compile-current-file)
      (condition-case err
          (require (quote foo) nil t)
        (error (message "Error requiring %s: %s" (quote foo) err) nil))))
  (if (and t t)
      (use-package-with-elapsed-timer "Loading package foo"
        (if (not (require (quote foo) nil t))
            (message "Could not load package %s" "foo")
          nil
          (quote foo-init)
          (quote foo-config)
          t))))
```